zerver/webhooks/honeybadger/doc.md

```diff
@@ -0,0 +1,27 @@
+{!create-stream.md!}
+
+{!create-bot-construct-url.md!}
+
+Honeybadger will use the `Honeybadger` stream by default if no stream is given
+in the URL. Make sure the selected stream exists in Zulip.
+
+**Follow these steps:**
+
+1. Go to `Alerts & Integrations` page of your `Honeybadger` Console.
+![](/static/images/integrations/honeybadger/002.png)
+
+2. Find and select the `WebHook` integration option.
+![](/static/images/integrations/honeybadger/003.png)
+
+3. Enter the URL constructed above into the URL field and configure
+the triggers for the webhook.
+![](/static/images/integrations/honeybadger/004.png)
+
+4. Test the webhook by clicking `Test This Integration`.
```

Comment:
Screenshots shouldn't display any private tokens - even if you are fine with sharing them publicly. You could blur the key in this image, but I recommend removing it completely, as it is not crucial for the process and there already are several other screenshots.